src/components/Error/Error.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect } from 'react';
+import './Error.scss';
+
+type Props = {
+  getDataError: boolean,
+  postDataError: boolean,
+  deleteDataError: boolean,
+  inputState: boolean,
+  disableErrorHandling: () => void,
+};
+
+export const Error: React.FC<Props> = ({
+  getDataError,
+  postDataError,
+  deleteDataError,
+  inputState,
+  disableErrorHandling,
+}) => {
+  const [timerId, setTimerId] = useState<NodeJS.Timeout | null>(null);
```

Comment:
It's better to use [refs](https://react.dev/reference/react/useRef) for timers. When a timer is stored in the state, any update to the timer value would trigger a re-render of the component.

Reply:
Fixed